sources/packages/web/src/views/Student.vue

```diff
@@ -179,17 +176,30 @@ export default {
     },
   },
   setup(props: any) {
+    // Readonly student data from state.
+    const store = useStore();
     const router = useRouter();
     const toast = useToast();
     const readonlyProfile = ref({});
+    const birthDate = ref();
```

Comment:
Is there a reason to birthDate and givennames, even being part of the readonlyProfile, have separated objects returned on setup to bind the UI?
My understanding is that readonlyProfile would be populated from the state or from DB, but their bind to the UI would be done using only readonlyProfile for the read-only fields.

Reply:
The state doesn't have the field First Name, it has GivenNames as coming from Keycloak but in our db we save First name. Does this explain? if not we can have a quick chat on Monday

Reply:
We can have a chat about this. The comment was not about the variables naming. The question is why birthDate and givennames (or first name) should be returned as separated objects in the setup.